balance/src/lib.rs

```diff
@@ -0,0 +1,350 @@
+extern crate futures;
+extern crate h2;
+extern crate http;
+extern crate tower_balance;
+extern crate tower_h2;
+
+use futures::{Async, Poll};
+use std::marker::PhantomData;
+use tower_balance::load::Instrument;
+
+/// Instruments HTTP responses to drop handles when their first body message is received.
+#[derive(Clone, Debug)]
+pub struct PendingUntilFirstData;
```

Comment:
Always gotta add private fields in public structs for forward compat. A single () field works.

Reply:
@carllerche Constructed via `Default`?